contracts/mocks/MockMarginCalculator.sol

```diff
@@ -14,6 +15,61 @@ contract MockMarginCalculator {
         view
         returns (bool isValid)
     {
+        if (finalVault.shortOtokens.length == 0 || finalVault.shortAmounts.length == 0) return true;
+
+        bool isExcess = getExcessMargin(finalVault, OtokenInterface(finalVault.shortOtokens[0]).collateralAsset());
+
+        return isExcess;
+    }
+
+    function getExcessMargin(MarginAccount.Vault memory _vault, address _denominated)
+        public
+        view
+        returns (bool isExcess)
+    {
+        // ensure the number of collateral, long and short array is valid.
+        _checkIsValidSpread(_vault);
+        // ensure the long asset is valid for the short asset.
+        _checkIsMarginableLong(_vault);
+
         return true;
```

Comment:
Mock this value instead of always using true

Reply:
yeah, for now will keep it as it is as I'm mocking a true result, will update it in the mint otoken PR, in either way this function will always return a predefined result(will not external call to oracle...etc)